crates/bevy_render/src/camera/camera.rs

```diff
@@ -193,6 +193,11 @@ impl Camera {
 
     /// Given a position in world space, use the camera to compute the viewport-space coordinates.
     ///
+    /// Note that the [`GlobalTransform`] can be out of sync with the [`Transform`] when
+    /// you modify the [`Transform`] of the camera in a system. When it seems the result of this function
+    /// lags behind the actual position make sure to run the system using this function in the [`CoreStage::PostUpdate`]
```

Comment:
I don't think this framing is correct. IMO, we should call out that you need to run this after the appropriate system (and link it).

Then, explain that "by default, this runs in...".